$NetBSD: patch-ab,v 1.2.8.1 2014/11/03 19:39:08 tron Exp $

- Fix build under Mac OS X (reorder os.path.join(ROOT,'include') )

--- setup.py.orig	2014-10-16 07:35:21.000000000 +0000
+++ setup.py
@@ -56,10 +56,10 @@ else:
 # - iconv.h
 # - libxslt/xsltconfig.h
 includes_dir = [
+os.path.join(ROOT,'include'),
 "/usr/include",
 "/usr/local/include",
 "/opt/include",
-os.path.join(ROOT,'include'),
 HOME
 ];
 
